CONTRIBUTING.md

```diff
@@ -1,41 +1,37 @@
 # Contributing
 
-When contributing to this repository, please first discuss the change you wish
-to make via issue, email, or any other method with the owners of this
-repository before making a change.
-
-Please note we have a code of conduct, please follow it in all your
-interactions with the project.
-
-## Pull Request Process
-
-1. Ensure any install or build dependencies are removed before the end of the
-layer when doing a build.
-2. Update the README.md with details of changes to the interface, this includes
-new environment variables, exposed ports, useful file locations and container
-parameters.
-3. Increase the version numbers in any examples files and the README.md to the
-new version that this Pull Request would represent. The versioning scheme we
-use is [SemVer](http://semver.org/).
-4. You may merge the Pull Request in once you have the sign-off of two other
-developers, or if you do not have permission to do that, you may request the
-second reviewer to merge it for you.
-
-## Code of Conduct
-
-### Our Pledge
-
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age,
-body size, disability, ethnicity, gender identity and expression, level of
-experience, nationality, personal appearance, race, religion, or sexual
+First of all, thanks for taking the time to contribute! 
+
+This file includes best practices and guidelines for contributing to our project. We encourage you to follow them and help us create a better product!
+
+## Best practices
+
+- We follow a [code of conduct](#code-of-conduct), review it and honor it in all your interactions with the project.
+
+- When contributing to this repository, first create an issue to explain the change you want to make.
+   - The issues will be preferably written in English, although Spanish is also acceptable.
+   - The issue title should briefly summarize the proposal.
+   - The issue body should include a description of the proposal with all the details, conditions, dependencies affected, charts, screenshots, etc., that can help to further understand and evaluate the proposal. 
+- Create a new branch before making any change in the repository. This branch should follow this format: #issue_ID_and_summary (for example, #23_add_new_provider).
+- Create commit messages that briefly describe the change applied, preferably in English.
```

Comment:
Please, make the unordered list marks uniform for the whole document. This list is using dashes `-` and the rest of them are using asterisks `*`.

Reply:
I have changed the others to "-" instead.